pkgs/top-level/all-packages.nix

```diff
@@ -12068,8 +12070,17 @@ in
 
   altcoins = recurseIntoAttrs ( callPackage ../applications/altcoins {
     callPackage = newScope { boost = boost155; };
-  } );
+  } ) // (
+    let callPackage = newScope { boost = boost162; openssl = openssl_1_1_0; };
```

Comment:
If it really wants particular versions of boost and openssl, wouldn't it be easier to just use those parameter names in the expression?